assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -28,9 +33,22 @@ import OptionalSettingsView from './OptionalSettingsView';
 import { MODULES_ANALYTICS } from '../../datastore/constants';
 import { MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
 import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import SettingsNotice, {
+	TYPE_WARNING,
+} from '../../../../components/SettingsNotice';
+import Link from '../../../../components/Link';
+import { useFeature } from '../../../../hooks/useFeature';
 const { useSelect } = Data;
 
 export default function SettingsView() {
+	const ga4Enabled = useFeature( 'ga4Reporting' );
```

Comment:
Again, please rename this for consistency:
```suggestion
	const ga4ReportingEnabled = useFeature( 'ga4Reporting' );
```

Reply:
Updated.